packages/core/src/utils.ts

```diff
@@ -228,7 +228,7 @@ export const getGasPriceOverrides = async (
     // On Gnosis, set the gas limit artificially high (since ethers does not seem to always estimate it proplerly especially for contract deployments)
     case 100:
     case 10200:
-      overridden.gasLimit = getMaxGasLimit(block.gasLimit)
+      overridden.gasLimit = (block.gasLimit / BigInt(4)) * BigInt(3)
```

Comment:
Just fixed this since it was convenient, don't want us to forget.

Reply:
Thanks. Just so we have some written record of why we set this to 3/4 of the block gas limit instead of 1/2: 

Say our batching logic (which uses heuristics) results in a batch that's 50% of the block gas limit. If the heuristic underestimates the gas used (i.e. the actual gas used is greater than 50%), we'd underfund the transaction by setting the `gasLimit` to be 50% of the block gas limit here.